### DISCUSSION TITLE
OpenCL Cmake option

### DISCUSSION THREAD

**Discussion by Vyrgyl** [pocl/pocl#1508] [Q&A]
Hi,
Is there a cmake option to force OpenCL version ? I would prefer 2.0 or 1.2 as I have older GPU.
Thanks,
Vyrgyl

**Comment by pjaaskel** [comment]
Hi Vyrgyl, We are deprecating 2.0 and 1.2 and switching to 3.0 by default since it has most 2.x features optional anyhow. Is there a reason not to support your GPU with 3.0 minimal profile?
